packages/core/utils/src/totals/__tests__/totals.ts

```diff
@@ -595,51 +594,50 @@ describe("Total calculation", function () {
           unit_price: 60,
           quantity: 2,
           subtotal: 100,
-          tax_total: 18.333333333333332,
-          total: 110,
+          tax_total: 18,
+          total: 108,
           is_tax_inclusive: true,
 
           original_total: 120,
           original_tax_total: 20,
 
-          discount_subtotal: 8.333333333333334,
-          discount_tax_total: 1.6666666666666667,
-          discount_total: 10,
+          discount_subtotal: 10,
+          discount_tax_total: 2,
+          discount_total: 12,
 
           tax_lines: [
             {
               rate: 20,
-              total: 18.333333333333332,
+              total: 18,
               subtotal: 20,
             },
           ],
           adjustments: [
             {
-              is_tax_inclusive: true,
-              amount: 10, // <- amount is tax inclusive so it's equal to total
-              subtotal: 8.333333333333334,
-              total: 10,
+              amount: 10,
+              subtotal: 10,
+              total: 12,
             },
           ],
         },
       ],
       subtotal: 100,
-      tax_total: 18.333333333333332,
-      total: 110, // total is 120 - 10 tax inclusive discount
```

Comment:
todo: this changed the point of this test, if you look at the comment above the test, we have:
- 120$ original total
- 10$ fixed discount 
- tax inclusive context & promotion

So we want the total to be 120$ - 10$ = 110$ here.